src/cpu/aarch64/jit_uni_binary.cpp

```diff
@@ -261,7 +262,8 @@ bool jit_uni_binary_t::pd_t::check_scales_mask() const {
     for (const auto &s : attr()->scales_.scales_) {
         if (s.second.mask_ != 0) return false;
     }
```

Comment:
```suggestion
```
This code is not needed, because the same checks are in the `attr_scales_ok`

Reply:
Thanks, this code has been removed now.